WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.m

```diff
@@ -379,7 +409,7 @@ - (BOOL)fb_dismissKeyboardWithKeyNames:(nullable NSArray<NSString *> *)keyNames
     id extractedElement = extractIssueProperty(issue, @"element");
     
     id<FBXCElementSnapshot> elementSnapshot = [extractedElement fb_takeSnapshot];
-    NSDictionary *elementAttributes = elementSnapshot ? [self.class dictionaryForElement:elementSnapshot recursive:NO] : @{};
+    NSDictionary *elementAttributes = elementSnapshot ? [self.class dictionaryForElement:elementSnapshot recursive:NO excludedAttributes:nil] : @{};
```

Comment:
Please reformat this ternary operator same way it was done below

Reply:
done